### ISSUE TITLE
Store security question answers as well as passwords

### ISSUE THREAD

**Issue by maxinfet** [bitwarden/core#24]
I would like to request the feature to store security questions as well as password. Many sites I visit now ask for security questions and I can never remember if I used upper case or lower case to answer the security questions. It would be great if bit warden could also store the questions and answers even if it didnt fill in the fields with them.

**Comment by kspearrin** [comment]
You can use the notes field for these types of things.

**Comment by maxinfet** [comment]
@kspearrin  I didn't even notice that till you mentioned it. I will go ahead and close this then. Thank you.

**Comment by zakpatterson** [comment]
Hey sorry to comment on such an old issue, but I love this issue. I've just started to use Bitwarden today. For years I have manually kept a long text file full of security questions and answers. Even one of my bank accounts will still allow one to reset my password using simple things such as my first pet's name and my city of birth. There's no reason these security questions couldn't be treated as auxiliary password fields for which random strings could be generated and stored.
